platform/android/MapboxGLAndroidSDKTestApp/src/test/java/com/mapbox/mapboxsdk/geometry/LatLngTest.java

```diff
@@ -202,4 +200,11 @@ public void testDescribeContents() {
         assertEquals("contents should be 0", 0, latLng.describeContents(), DELTA);
     }
 
+    @Test
+    public void testWrapped() {
+        LatLng latLng = new LatLng(45.0, -185.0);
+        LatLng wrapped = latLng.wrap();
+        assertTrue(!latLng.equals(wrapped));
+        assertTrue(wrapped.getLongitude() == 175.0);
```

Comment:
For consistency, this should be `assertEquals("longitude wrapped value", wrapped.getLongitude(), 175.0, DELTA);`